### ISSUE TITLE
Setting option disable_xcpretty to true crashes multi_scan's retry functionality

### ISSUE THREAD

**Issue by jambudi** [lyndsey-ferguson/fastlane-plugin-test_center#275]
All tests are executing as per expectation and at the end, We are seeing this error 

+----------```
--------------------------------------+---------------------------------------------------------------------------------------+
|                                                        Summary for scan 2.153.1                                                        |
+------------------------------------------------+---------------------------------------------------------------------------------------+
| workspace                                      | /****.xcworkspace |
| only_testing                                   | ["/****/"]                                                                |
| app_identifier                                 | /****/                                                                      |
| scheme                                         | /****/                                                                            |
| devices                                        | ["iPhone 11 Pro (13.3)"]                                                              |
| disable_concurrent_testing                     | true                                                                                  |
| disable_xcpretty                               | true                                                                                  |
| output_types                                   | junit                                                                                 |
| output_files                                   | report.junit                                                                          |
| code_coverage                                  | false                                                                                 |
| derived_data_path                              | ./fastlane/DerivedData                                                                |
| reinstall_app                                  | true                                                                                  |
| reset_simulator                                | true                                                                                  |
| fail_build                                     | false                                                                                 |
| xcargs                                         | -quiet                                                                                |
| buildlog_path                                  | ./fastlane/test_output/smoke_results                                                  |
| output_directory                               | ./fastlane/test_output/smoke_results                                                  |
| skip_detect_devices                            | false                                                                                 |
| force_quit_simulator                           | false                                                                                 |
| disable_slide_to_type                          | true                                                                                  |
| clean                                          | false                                                                                 |
| open_report                                    | false                                                                                 |
| should_zip_build_products                      | false                                                                                 |
| use_clang_report_name                          | false                                                                                 |
| build_for_testing                              | true                                                                                  |
| slack_use_webhook_configured_username_and_icon | false                                                                                 |
| slack_username                                 | fastlane                                                                              |
| slack_icon_url                                 | https://fastlane.tools/assets/img/fastlane_icon.png                                   |
| skip_slack                                     | false                                                                                 |
| slack_only_on_failure                          | false                                                                                 |
| xcodebuild_command                             | env NSUnbufferedIO=YES xcodebuild                                                     |
| include_simulator_logs                         | false                                                                                 |
| skip_build                                     | false                                                                                 |
| xcode_path                                     | /Applications/Xcode.app                                                               |
+------------------------------------------------+---------------------------------------------------------------------------------------+
```

DEBUG [2020-08-23 10:56:17.14]: Before building, removing pre-existing xctestrun files: ["./fastlane/DerivedData/Build/Products//****_iphonesimulator13.2-x86_64.xctestrun"]
DEBUG [2020-08-23 10:56:17.14]: Fetching available simulator devices
INFO [2020-08-23 10:56:17.36]: Resetting iPhone 11 Pro
DEBUG [2020-08-23 10:56:17.63]: Fetching available simulator devices
INFO [2020-08-23 10:56:17.86]: Resetting iPhone 11 Pro
DEBUG [2020-08-23 10:56:18.08]: Fetching available simulator devices
INFO [2020-08-23 10:56:18.35]: Disabling 'Slide to Type' iPhone 11 Pro
INFO [2020-08-23 10:56:18.35]: $ /usr/libexec/PlistBuddy -c "Add :KeyboardContinuousPathEnabled bool false" /Users//****//Library/Developer/CoreSimulator/Devices/4620252A-4855-4102-AD4C-327F98256330/data/Library/Preferences/com.apple.keyboard.ContinuousPath.plist >/dev/null 2>&1
DEBUG [2020-08-23 10:56:18.36]: Fetching available simulator devices
INFO [2020-08-23 10:56:18.62]: Disabling 'Slide to Type' iPhone 11 Pro
INFO [2020-08-23 10:56:18.62]: $ /usr/libexec/PlistBuddy -c "Add :KeyboardContinuousPathEnabled bool false" /Users//****//Library/Developer/CoreSimulator/Devices/D1A6E91A-0761-40C6-A806-09D7DBF5AD29/data/Library/Preferences/com.apple.keyboard.ContinuousPath.plist >/dev/null 2>&1
DEBUG [2020-08-23 10:56:18.63]: Uninstalling app '****' from iPhone 11 Pro...
INFO [2020-08-23 10:56:18.63]: Launch Simulator iPhone 11 Pro
INFO [2020-08-23 10:56:18.63]: $ xcrun instruments -w 4620252A-4855-4102-AD4C-327F98256330 &> /dev/null
INFO [2020-08-23 10:56:25.97]: Uninstall application ****
INFO [2020-08-23 10:56:25.97]: $ xcrun simctl uninstall 4620252A-4855-4102-AD4C-327F98256330 **** &> /dev/null
DEBUG [2020-08-23 10:57:08.65]: Uninstalling app '****' from iPhone 11 Pro...
INFO [2020-08-23 10:57:08.65]: Launch Simulator iPhone 11 Pro
INFO [2020-08-23 10:57:08.65]: $ xcrun instruments -w D1A6E91A-0761-40C6-A806-09D7DBF5AD29 &> /dev/null
INFO [2020-08-23 10:57:16.77]: Uninstall application ****
INFO [2020-08-23 10:57:16.77]: $ xcrun simctl uninstall D1A6E91A-0761-40C6-A806-09D7DBF5AD29 **** &> /dev/null
INFO [2020-08-23 10:57:41.75]: $ set -o pipefail && env NSUnbufferedIO=YES xcodebuild -workspace /Users/***.xcworkspace -scheme *** -destination 'platform=iOS Simulator,id=4620252A-4855-4102-AD4C-327F98256330' -destination 'platform=iOS Simulator,id=D1A6E91A-0761-40C6-A806-09D7DBF5AD29' -derivedDataPath './fastlane/DerivedData' -disable-concurrent-testing -enableCodeCoverage NO -quiet -only-testing:*** build-for-testing | tee '***.log'


10:36:37  [worker 3] [22:06:11]: ------------------------------
10:36:37  [worker 3] [22:06:11]: --- Collecting system logs ---
10:36:37  [worker 3] [22:06:11]: ------------------------------
10:36:37  [worker 3] [22:06:11]: $ xcrun simctl spawn D2427E9D-B344-4208-A5F2-32CA90333544 log collect --output /***/fastlane/test_output/smoke_results/***-batch-3/system_logs-iPhone\ 11\ Pro\ Clone\ 3\ TestCenter::Helper::MultiScanManager::SimulatorHelper\<70206711797300\>_iOS_13.3.logarchive 2>/dev/null

10:36:37  [worker 3] [22:06:13]: Error setting value '***/fastlane/test_output/smoke_results/ApplicationUITests-batch-3/report.junit' for option 'junit'

10:36:37  [worker 3] Error: cannot find the junit xml report file '***/fastlane/test_output/smoke_results/ApplicationUITests-batch-3/report.junit'

10:36:37  [worker 3] ["***/vendor/bundle/ruby/2.6.0/gems/fastlane-2.153.1/fastlane_core/lib/fastlane_core/configuration/config_item.rb:224:in `rescue in valid?'", "/private/var/jenkins/workspace/***

2028/vendor/bundle/ruby/2.6.0/gems/fastlane-2.153.1/fastlane_core/lib/fastlane_core/configuration/config_item.rb:220:in `valid?'", "/private/var/jenkins/workspace/***/vendor/bundle/ruby/2.6.0/gems/fastlane-

2.153.1/fastlane_core/lib/fastlane_core/configuration/config_item.rb:182:in `verify!'", "/private/var/jenkins/workspace/***vendor/bundle/ruby/2.6.0/gems/fastlane-

2.153.1/fastlane_core/lib/fastlane_core/configuration/configuration.rb:87:in `block in verify_value_exists'", "/private/var/jenkins/workspace/***/vendor/bundle/ruby/2.6.0/gems/fastlane-


**At end getting this error** 

10:36:37  +------------------+-----------------+
10:36:37  |            Lane Context            |
10:36:37  +------------------+-----------------+
10:36:37  | DEFAULT_PLATFORM | ios             |
10:36:37  | PLATFORM_NAME    | ios             |
10:36:37  | LANE_NAME        | ios smoke_tests |
10:36:37  +------------------+-----------------+
10:36:37  [22:06:14]: No such file or directory @ rb_file_s_stat - /private/var/jenkins/workspace/***/fastlane/test_output/smoke_results/ApplicationUITests/.

**Comment by jambudi** [comment]
After `disable_xcpretty: false`, Jenkins's console output is not formatted.  Do you have any suggestions on how to print console output formatted?

**Comment by lyndsey-ferguson** [comment]
@jambudi can you elaborate? I never use that flag and the Jenkins console output seems normal (?). What are the differences that you see?

**Comment by jambudi** [comment]
@lyndsey-ferguson  I think everything looks good.  After changing from true to false for `disable_xcpretty: false`. All test results are merged and working as expected. thank you

**Comment by lyndsey-ferguson** [comment]
Great, thank you! I will change this Issue title so that I prevent that from happening in the action in itself

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.14.2](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.14.2)
